web-api/terraform/modules/lambda-role/lambda-role.tf

```diff
@@ -78,6 +78,20 @@ resource "aws_iam_role_policy" "lambda_policy" {
             ],
             "Effect": "Allow"
         },
+        {
+            "Action": [
+                "ec2:DescribeInstances",
```

Comment:
In aws are the permissions for rds actually under ec2? Why do we need ec2:DescribeInstances permission?

Reply:
Maybe a leftover from the earlier VPN approach?